bpf/go_common.h

```diff
@@ -21,11 +21,13 @@
 #include "tracing.h"
 #include "trace_util.h"
 #include "go_offsets.h"
-#include "go_traceparent.h"
 #include "pin_internal.h"
 
 char __license[] SEC("license") = "Dual MIT/GPL";
 
+#define W3C_KEY_LENGTH 11
+#define W3C_VAL_LENGTH 55
```

Comment:
```suggestion
enum : u8 {
  W3C_KEY_LENGTH = 11u,
  W3C_VAL_LENGTH = 55u
};
```

rationale: _**in theory**_, this provides enhanced type-safety as it's handled by the compiler rather than the pre-processor. Obscure bugs such as macro redefinitions get caught by the compiler and an error is emitted.

It will always fit right with the types of `grpc_header_field` which expect u64.

Having said that, if W3C_KEY_LENGTH is meant to be the lenght of the `"traceparent"` literal, it may be clearer (and less error-prone) to write:

```
static const char W3C_KEY[] = "traceparent";
static const size_t W3C_KEY_LENGTH = sizeof(W3C_KEY) - 1;
```

and you can then reuse the `W3C_KEY` constant where the `"traceparent"` literal is being used in the code.